osmomath/binary_search.go

```diff
@@ -141,6 +141,23 @@ func (e ErrTolerance) CompareBigDec(expected BigDec, actual BigDec) int {
 	return 0
 }
 
+// EqualCoins returns true iff the two coins are equal within the ErrTolerance constraints and false otherwise.
+// TODO: move error tolerance functions to a separate file.
+func (e ErrTolerance) EqualCoins(expectedCoins sdk.Coins, actualCoins sdk.Coins) bool {
+	if len(expectedCoins) < len(actualCoins) {
+		return false
+	}
+
+	for _, expectedCoin := range expectedCoins {
+		curCoinEqual := e.Compare(expectedCoin.Amount, actualCoins.AmountOf(expectedCoin.Denom))
+		if curCoinEqual != 0 {
+			return false
+		}
+	}
+
+	return true
+}
+
```

Comment:
Im confused how this PR is passing when the go.mod wasn't updated to import this new osmomath method

Reply:
Hmm yeah this is actually odd

Reply:
Our go work uses local wiring for tests, its just e2e that and live code build that would have such an issue. Since this is test only code, theres no issue.